content/docs/forms.md

```diff
@@ -0,0 +1,214 @@
+| `fields`            | An array of fields that will define the shape of the form and how content is edited.                                                          |
+| `loadInitialValues` | _Optional:_ A function to load the initial form state asynchronously. Return a promise that passes an object of form values when it resolves. |
+| `reset`             | _Optional:_ A function that runs when the form state is reset by the user.                                                                    |
+| `actions`           | _Optional:_ An array of custom actions that will be added to the form.                                                                        |
+| `meta`              | _Optional:_ Arbitrary key-value pairs for your internal use.                                                                                  |
+| `__type`            | _Optional:_ Sets the Form's plugin type. Automatically set based on which form hook is used.                                                  |
+
+Now that we know how to configure a form, let's revisit the simplified example from the beginning of this document to demonstrate how we might configure this form:
+
+```javascript
+import * as React from React
+import ReactMarkdown from 'react-markdown'
+import { useLocalForm } from 'tinacms'
+
+export function Page(props) {
+    const [modifiedValues] = useLocalForm({
+      id: props.fileRelativePath,
+      label: 'Edit Post',
+      fields: [
+        {
+          name: 'title',
+          label: 'Title',
+          component: 'text',
+        },
+        {
+          name: 'markdownContent',
+          label: 'content',
+          component: 'markdown',
+        }
+      ],
+      initialValues: {
+        title: props.title,
+        markdownContent: props.markdownContent
+      },
+      onSubmit: (formData) => {
+        // save the new form data
+      },
+    })
+    return (
+        <main>
+            <h1>{modifiedValues.title}</h1>
+            <ReactMarkdown source={modifiedValues.markdownContent}>
+        </main>
+    )
+}
+
+```
+
+> Note that when using these basic hooks, you are expected to implement the save functionality yourself by adding an `onSubmit` function. By default, Tina makes no assumptions about how your content is stored. These basic form hooks are building blocks for creating more purpose-built tools to fit specific use cases.
+
+### Watched Vars: Preserve Form Reactivity
+
+The second argument that can be passed to the form hooks is an optional object containing values that the form will need to react to. Use this when dealing with data from external sources or in an environment with Hot Module Replacement; in other words, when you expect this data to change and it's essential to keep the form in sync with it.
+
+```typescript
+interface WatchableFormValue {
+  values: any
+  label: FormOptions<any>['label']
+  fields: FormOptions<any>['fields']
```

Comment:
Duplication in the docs is okay. Makes things more readable.

```suggestion
  label: string
  fields: Field[]
```

Reply:
for sure, this is better. I just copypasted these interfaces from the source